### PR TITLE
update to 2.7.0

### PR DESCRIPTION
**Pull request by cbouss**
ChangeS:
- Update to 2.7.0

https://github.com/aio-libs/aiobotocore/tree/2.7.0

### CODE REVIEW

**Review by ryanskeith — APPROVED** [review]
LGTM

File: recipe/meta.yaml [hunk 1]
```diff
@@ -21,11 +21,11 @@ requirements:
...
   run:
-    - aiohttp >=3.3.1
-    - aioitertools >=0.5.1
-    - botocore >=1.29.76,<1.29.77
+    - aiohttp >=3.7.4.post0,<4.0.0
+    - aioitertools >=0.5.1,<1.0.0
+    - botocore >=1.31.16,<1.31.65
     - python
-    - wrapt >=1.10.10
+    - wrapt >=1.10.10,<2.0.0
```
Comment (M-Waszkiewicz-Anaconda) [inline recipe/meta.yaml:31]: maybe 
run _constrains: 
- boto3 >=1.28.16,<1.28.65
see, https://github.com/aio-libs/aiobotocore/blob/9c6db8a9fd12a244a0b7413fd182857e603d08f9/setup.py#L18

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM, maybe boto3 in run_constrains ?

File: recipe/meta.yaml (hunk 1 above)
Comment (cbouss) [reply recipe/meta.yaml:31]: thanks

**Pull request merged**